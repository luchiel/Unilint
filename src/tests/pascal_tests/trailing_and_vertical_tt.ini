[other]
forbid_vertical_alignment = true
forbid_trailing_whitespaces = true
