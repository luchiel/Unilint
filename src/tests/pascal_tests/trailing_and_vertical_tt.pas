var
  c: char;
begin      
  case c of
    'a':  b;
    'b':  c;
  end;
end.