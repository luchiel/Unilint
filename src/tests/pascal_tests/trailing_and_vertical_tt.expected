3:6: trailing whitespaces
5:9: more than one whitespace
6:9: more than one whitespace
